### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by monkeyskullfire** [MicrosoftDocs/sccm-docs-powershell-ref#72]
Submitting this per conversation with Aaron at MMSMOA.  This link no longer works:  https://docs.microsoft.com/en-us/powershell/module/configurationmanager/about_configurationmanager_cmdlets?view=sccm-ps

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e2f0b6d-c7ca-cb30-3de9-e50eae55c24d
* Version Independent ID: 01e64fb6-4385-6864-7430-38ac98632aaf
* Content: [ConfigurationManager](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/?view=sccm-ps)
* Content Source: [sccm-ps/ConfigurationManager/ConfigurationManager.md](https://github.com/MicrosoftDocs/sccm-docs-powershell-ref/blob/master/sccm-ps/ConfigurationManager/ConfigurationManager.md)
* Service: **powershell**

**Comment by aczechowski** [comment]
Thanks @monkeyskullfire for reminding me 😄 
I looked into it, I deleted the file a few weeks ago. It was poorly formatted and basically a duplicate of the [overview](https://docs.microsoft.com/powershell/sccm/overview?view=sccm-ps) article. At the time I forgot to create a redirect, but I'm adding one now. It should go live later this afternoon. I'll keep this open until I can confirm (in part to make sure I got the redirect syntax correct!)
